src/react-mf-styleguide.js

```diff
@@ -1,4 +1,7 @@
 import "./set-public-path";
 import "./global.css?modules=false";
+import _Button from "./button.component.js";
 
 // You can also export React components from this file and import them into your microfrontends
+
+export const Button = _Button;
```

Comment:
I initially was going to use https://babeljs.io/docs/en/babel-plugin-proposal-export-default-from but I was worried it would make the example paradoxically more complex because we'd have to add more custom rules for eslint

Reply:
Can't you do `export { default as Button } from './button.component.js';`?? That's what we do at midigator and it works - no special babel plugin needed.

Reply:
That worked well